packages/block-editor/src/components/index.js

```diff
@@ -126,10 +126,7 @@ export { default as BlockSettingsMenuControls } from './block-settings-menu-cont
 export { default as BlockTitle } from './block-title';
 export { default as BlockToolbar } from './block-toolbar';
 export { default as BlockTools } from './block-tools';
-export {
-	default as CopyHandler,
-	useClipboardHandler as __unstableUseClipboardHandler,
```

Comment:
Isn't this a breaking change? should we continue exporting this.

Reply:
Restored it